### ISSUE TITLE
Related to junction

### ISSUE THREAD

**Issue by nowStart9426** [daohu527/imap#78]
For example, I have a junction with ID 1 that generates a statement similar to the following after transformation. Is this expected？

`
<junction name="" id="1" type="default">

  successor_id {
    id: "road_1_lane_0_-1"
  }
`

**Comment by daohu527** [comment]
In fact, I did not understand your question, we combine the names according to the road and lane id, which does not involve the junction id.

**Comment by nowStart9426** [comment]
Yes, I also find it difficult to understand.. Please allow me to describe my problem again.. Let me show you the code content below
1、This is the corresponding Juncton content
`<junction name="" id="1285" type="default">
 <connection id="0"        incomingRoad="100000"           connectingRoad="100032" contactPoint="start">
<laneLink from="-1" to="-1"/>
<laneLink from="-2" to="-2"/>
<laneLink from="-3" to="-3"/>
<laneLink from="-4" to="-4"/>
</connection>
 <connection id="1"        incomingRoad="100000"           connectingRoad="100033" contactPoint="start">
<laneLink from="-1" to="-1"/>
<laneLink from="-2" to="-2"/>
<laneLink from="-3" to="-3"/>
</connection>
 <connection id="2"        incomingRoad="100003"           connectingRoad="100030" contactPoint="end">
<laneLink from="-1" to="-1"/>
<laneLink from="-2" to="-2"/>
</connection>
 <connection id="3"        incomingRoad="100003"           connectingRoad="100037" contactPoint="end">
<laneLink from="-1" to="-1"/>
<laneLink from="-2" to="-2"/>
</connection>
 <connection id="4"        incomingRoad="100003"           connectingRoad="100031" contactPoint="end">
<laneLink from="-1" to="-1"/>
<laneLink from="-2" to="-2"/>
<laneLink from="-3" to="-3"/>
<laneLink from="-4" to="-4"/>
</connection>
</junction>`
2、Next is the generated content after conversion. there are eight predecessors and successors, they belong to the road_ 100000 and road_100003
` successor_id {
    id: "road_1285_lane_0_-1"
  predecessor_id {
    id: "road_1285_lane_0_-2"
  }
 predecessor_id {
    id: "road_1285_lane_0_-3"
  }
 predecessor_id {
    id: "road_1285_lane_0_-4"
  }
  }`

**Comment by MrGoldminer** [comment]
this is also my questions，so now have a way to fix this problems?

**Comment by MrGoldminer** [comment]
apollo_routing.map里面edge消息会存在这些内容
edge {
  from_lane_id: "lane_56"
  to_lane_id: "lane_27"
  cost: 0
  direction_type: FORWARD
}
edge {
  from_lane_id: "lane_57"
  to_lane_id: "lane_21"
  cost: 0
  direction_type: FORWARD
}
edge {
  from_lane_id: "lane_58"
  to_lane_id: "lane_24"
  cost: 0
  direction_type: FORWARD
}
